lib/Containers/SmallVector.h

```diff
@@ -23,15 +23,22 @@
 
 #pragma once
 
+#include <algorithm>
 #include <vector>
 
 #include "Containers/details/short_alloc.h"
 
-namespace arangodb {
-namespace containers {
+namespace arangodb::containers {
+
+#ifndef _DEBUG
+constexpr static auto minAlignment = std::size_t{1};
+#else
+// Make Debug-MSVC happy
+constexpr static auto minAlignment = alignof(std::_Container_proxy);
```

Comment:
Is `std::_Container_proxy` available on all platforms when `_DEBUG` is defined?
Should it be wrapped into another ifdef that checks for MSVC?

Reply:
I assume that it's only set for MSVC, as it's a flag used in MSVC's STL specifically, afaik. However, it still seems prudent to check for both MSVC and _DEBUG; I will add it.